lib/honeybadger/agent.rb

```diff
@@ -130,6 +134,10 @@ def notify(exception_or_opts, opts = {})
         with_error_handling { hook.call(notice) }
       end
 
+      # Clean breadcrumbs after hooks (as they could have been mutated)
```

Comment:
Just curious, where can the breadcrumbs be mutated before reaching the notify method?

Reply:
For the active_support notifications, they can use the `:transform` key in the configuration for each type of notification. There currently is not a way for them to mutate any of the logging events before notice. They could also always mutate at `Agent.breadcrumbs` at any time if they wanted.